src/QuickMarcEditor/QuickMarcEditWrapper.test.js

```diff
@@ -408,7 +411,10 @@ describe('Given QuickMarcEditWrapper', () => {
 
         await fireEvent.click(getByText('stripes-acq-components.FormFooter.save'));
 
-        expect(mockShowCallout).toHaveBeenCalledWith({ messageId: 'ui-quick-marc.record.save.updated' });
+        expect(mutator.quickMarcEditInstance.GET).toHaveBeenCalled();
```

Comment:
Please also add a test, where `quickMarcEditInstance` returns a different record version, so we can verify that optimistic locking banner is shown